src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'under Construction',
+};
+
+const categories = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    setStatus: (state, action) => {
+      state.status = action.payload;
+    },
```

Comment:
As mentionned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer%20that%20checks%20the%20status%20and%20always%20returns%20%22Under%20construction%22%20(the%20initial%20state%20should%20check%20to%20that%20string)), this reducer is suposed to check the status, and always return a string that says `Under construction`.